internal/clients/diffreviewer/circleci/circleci_service.go

```diff
@@ -112,7 +118,7 @@ func (s *Service) LoadConfig(nightfallConfigFileName string) (*nightfallconfig.C
 	}
 
 	var maxNumberRoutines int
-	if nightfallConfig.MaxNumberRoutines < nightfall.MaxConcurrentRoutinesCap {
+	if nightfallConfig.MaxNumberRoutines < nightfall.MaxConcurrentRoutinesCap && nightfallConfig.MaxNumberRoutines > 0 {
```

Comment:
do we want to set the default to something lower than the max when there isn't one? How does 20 sound (`nightfall.MaxConcurrentRoutinesCap=50`)